go/sdk/v1/state/read.go

```diff
@@ -36,3 +37,19 @@ func ReadUint32(key []byte) uint32 {
 	}
 	return binary.LittleEndian.Uint32(bytes)
 }
+
+func ReadBytes20(key []byte) (out [20]byte) {
+	bytes := ReadBytes(key)
+	if len(bytes) < 20 {
+		return
+	}
+	return *(*[20]byte)(unsafe.Pointer(&bytes[0]))
+}
+
+func ReadBytes32(key []byte) (out [32]byte) {
+	bytes := ReadBytes(key)
+	if len(bytes) < 20 {
+		return
+	}
+	return *(*[32]byte)(unsafe.Pointer(&bytes[0]))
```

Comment:
Is there a way to avoid this? Even if it's a copy we can optimize it later when (if) it becomes a problem.